app/src/main/kotlin/com/exallium/todoapp/notedetail/NoteDetailPresenter.kt

```diff
@@ -21,14 +21,13 @@ class NoteDetailPresenter(private val view: NoteDetailView,
         lookupNoteDetail(noteId)
     }
 
-    fun lookupNoteDetail(noteId: String) {
-        compositeSubscription.add(model.getNoteById(noteId).subscribe(object : SingleSubscriber<Note>() {
+    private fun lookupNoteDetail(noteId: String) {
+        compositeSubscription.add(model.getNote(noteId).subscribe(object : SingleSubscriber<Note>() {
```

Comment:
Ok I removed the debounce functionality due to testing considerations. Added a separate ticket https://github.com/exallium/todoapp/issues/35